### PR TITLE
Facebook login

### LINKED ISSUE #6: Facebook-Login
As a user I want to login with Facebook.

### PR DESCRIPTION
**Pull request by ElisabethSeitz**

### CODE REVIEW

File: frontend/src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
-/*test('renders learn react link', () => {
+xtest('renders learn react link', () => {
```
Comment (fabianschmauder) [inline frontend/src/App.test.js]: i would fix this or write another test

Comment (fabianschmauder) [reply frontend/src/App.test.js]: maybe start with an easy on for your monster component

**Review by fabianschmauder — COMMENTED** [review]
good job 🎉

**Review by saskiabusch — APPROVED** [review]

**Review by saskiabusch — APPROVED** [review]
Loooks amazing

**Review by TobiasHasenkamp — APPROVED** [review]
test approve

**Review by TobiasHasenkamp — COMMENTED** [review]
test

**Review by kaiheinrich — APPROVED** [review]

**Pull request merged**